### DISCUSSION TITLE
Flockdrones deconstructing non-Flock stuff

### DISCUSSION THREAD

**Discussion by FlameArrow57** [stonepillars/goon-flock#229] [Ideas]
Discussion for Flockdrones being able to deconstruct non-Flock stuff. Discussion version of #170.

I think stuff like walls shouldn't be able to be deconstructed. Reinforces the idea that stuff needs to be converted before deconstructed. This can be avoided if having enough resources too. Not having enough resources would be considered something the Flockmind needs to think about beforehand, or a case where the Flock loses a battle. Kind of poses a restriction on humans too.

For machinery, I think it could be a good idea to help prevent exploiting by walling in Flock with stuff they can't move past, but that again kind of poses a restriction on humans. Maybe could implement it instead so that machinery on Flock converted tiles is destroyed by tile conversion. Resource cubes could be dropped, giving another source of resources, and like said already would prevent human stuff from being left on Flock converted tiles.

**Comment by amylizzle** [comment]
I think ideally there should be nothing the flock can't convert. It might not be a terrible idea to have a default that just colours the object, overlays it with some sparkly flock stuff, and throws the flockthing atom property on it.

**Reply by TobleroneSwordfish** [reply]
Would that prevent humans from using it?

**Comment by stonepillars** [comment]
During the playtest last week, flockdrones were able to break into human forts by converting the walls and busting them down. Seemed pretty good trade-off, since it required resources.

**Reply by TobleroneSwordfish** [reply]
![image](https://user-images.githubusercontent.com/20713227/167113859-343802a0-638a-4859-8769-6af6a5709fb7.png)
